datafusion/src/logical_plan/expr.rs

```diff
@@ -1360,24 +1360,63 @@ pub fn in_list(expr: Expr, list: Vec<Expr>, negated: bool) -> Expr {
 pub trait Literal {
     /// convert the value to a Literal expression
     fn lit(&self) -> Expr;
+
+    /// convert the number to literal expression of timestamp in nanosecond
+    fn lit_timestamp_nanosecond(&self) -> Expr;
```

Comment:
I was thinking perhaps calling this `lit_timestamp_nano` or something would be more consistent with chrono, but when I looked, `chrono` appears to be inconsistent on this point (uses both `nano` and `nanosecond`) https://docs.rs/chrono/0.4.19/chrono/?search=nano

Reply:
@alamb @Dandandan  and @houqp I have added a new trait for it. Can you have a look?